presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ImplementIntersectAndExceptAsUnion.java

```diff
@@ -97,6 +118,7 @@ public PlanNode optimize(PlanNode plan, Session session, Map<Symbol, Type> types
             extends SimplePlanRewriter<Void>
     {
         private static final String INTERSECT_MARKER = "intersect_marker";
+        private static final String EXCEPT_MARKER = "except_marker";
```

Comment:
There's nothing special about the name "intersect_marker", so we might as well just rename it to "marker" and use the same name for both cases. There's not much benefit in parameterizing it.